Kitodo/src/main/java/org/kitodo/production/forms/createprocess/ProcessFieldedMetadata.java

```diff
@@ -677,16 +674,27 @@ public void preserve() throws InvalidMetadataValueException, NoSuchMetadataField
                 division.setLabel(null);
             }
             metadata.clear();
+
             for (TreeNode child : treeNode.getChildren()) {
                 ProcessDetail row = (ProcessDetail) child.getData();
-                Pair<BiConsumer<Division<?>, String>, String> metsFieldValue = row.getStructureFieldValue();
-                if (Objects.nonNull(metsFieldValue)) {
-                    metsFieldValue.getKey().accept(division, metsFieldValue.getValue());
+                String id = row.getMetadataID();
+                if (row instanceof ProcessSimpleMetadata && specialFields.contains(id)
+                        && ((ProcessSimpleMetadata) row).getSettings()
+                        .getDomain().orElse(Domain.DESCRIPTION).equals(Domain.METS_DIV)) {
+                    updateDivisionFromProcessDetail(id, (ProcessSimpleMetadata) row);
                 } else {
                     metadata.addAll(row.getMetadataWithFilledValues());
                 }
             }
-            if (Objects.nonNull(hiddenMetadata)) {
+            if (!Objects.isNull(hiddenMetadata) && !hiddenMetadata.isEmpty()) {
```

Comment:
Just one very small nitpick: is there a reason why you replaced `Objects.nonNull` with `!Objects.isNull`? The later reads like a double negation to me and is just less readable than the dedicated `nonNull` check, IMHO.

Reply:
Good catch; i do not know. I replaced it with nonNull.